### PR TITLE
Register new snapshots

### PR DESCRIPTION
**Pull request by aturon**
r? @alexcrichton

### CODE REVIEW

File: src/liballoc/lib.rs [hunk 1]
```diff
@@ -70,7 +70,6 @@
        test(no_crate_inject, attr(allow(unused_variables), deny(warnings))))]
 #![no_std]
 #![needs_allocator]
-#![cfg_attr(not(stage0), deny(warnings))]
```
Comment (alexcrichton) [inline src/liballoc/lib.rs]: Ah can these actually stay? They're intended to emulate what the makefiles did with `-D warnings`.

Although I also personally `git grep stage0` to find what to remove on new snapshots. I would personally be in favor of changing this to `#![deny(warnings)]` (e.g. in all stages), but others may feel differently

File: src/liballoc/lib.rs [hunk 2]
```diff
@@ -92,16 +91,9 @@
...
 
-// Issue# 30592: Systematically use alloc_system during stage0 since jemalloc
-// might be unavailable or disabled
-#![cfg_attr(stage0, feature(alloc_system))]
-
 #![cfg_attr(not(test), feature(raw, fn_traits, placement_new_protocol))]
 #![cfg_attr(test, feature(test, rustc_private, box_heap))]
 
-#[cfg(stage0)]
-extern crate alloc_system;
```
Comment (alexcrichton) [inline src/liballoc/lib.rs]: Oh and these two comments I believe need to say here as well, although if you want to add docs to not delete them on snapshots I'd also be fine.

**Comment by alexcrichton** [comment]
just a nit about `#![cfg_attr(stage0, deny(warnings))]` but otherwise lgtm

File: src/liballoc/lib.rs (hunk 1 above)
Comment (aturon) [inline src/liballoc/lib.rs]: Oh, whoops, yes I was just doing a grep removal, but that doesn't make sense here. Will fix.

**Comment by alexcrichton** [comment]
cc @dhuseby, @semarie, @mneumann  this'll probably need some manual snapshot uploads for the BSDs

**Comment by semarie** [comment]
@alexcrichton here it is : http://semarie.online.fr/rust/rust-stage0-2016-02-18-4d3eebf-openbsd-x86_64-ac957c6b84de2bd67f01df085d9ea515f96e22f3.tar.bz2

It is 4d3eebf + cherry-pick of #31727

**Comment by alexcrichton** [comment]
Thanks @semarie! I corrected the date in the tarball name to `2016-02-17`, but other than that it's uploaded and should be ready to go. @aturon if this bounces, can you add this to src/snapshots.txt?

```
  openbsd-x86_64 ac957c6b84de2bd67f01df085d9ea515f96e22f3
```

**Comment by dotdash** [comment]
The bors failure is just a timeout, but the tidy errors on Travis need to be fixed I think?

**Comment by aturon** [comment]
Added the openbsd snap and fixed tidy error.

@bors: r=alexcrichton

**Comment by alexcrichton** [comment]
@bors: retry

Fixed in #31855. One thing I also notice, if this bounces again, could you also update `src/nightlies.txt`? The nightly version for rustbuild will also need to change eventually, but that can happen as a separate PR instead of this one.

**Comment by dhuseby** [comment]
@alexcrichton i'll make bitrig and freebsd snapshots soon.

**Comment by mneumann** [comment]
@dhuseby : Then please also add the Dragonfly (3391630) snapshot to src/snapshots.txt once you're done.

**Comment by dhuseby** [comment]
@mneumann i'll see if i can get one built soon.

**Comment by mneumann** [comment]
@dhuseby: I already built one and uploaded it. So you can add mine if you like :)

**Pull request merged**